### ISSUE TITLE
Git worktrees project detection failing

### ISSUE THREAD

**Issue by fazibear** [wakatime/wakatime-cli#1001]
When using worktrees project is incorrectly detected. Branch name is taken instead of project name. 

**Environment**:
- OS: mac
- Platform: arm64

Git setted like this:
```bash
git clone --bare https://github.com/wakatime/wakatime-cli wakatime-cli
cd wakatime-cli
git worktree add master
cd master
```

Also tired https://morgan.cugerone.com/blog/how-to-use-git-worktree-and-in-a-clean-way/ with .bare solution but with no luck.

Looks like it failing on this: https://github.com/wakatime/wakatime-cli/blob/develop/pkg/project/git.go#L247 (there is no dir .git)

**Comment by gandarez** [comment]
I can see it's not correctly getting project name, I'll debug and come back with a solution.

**Comment by gandarez** [comment]
@fazibear released version 1.88.1 to fix it.
